frontend/shoot-client/src/components/App/App.context.tsx

```diff
@@ -3,6 +3,10 @@ import React, { createContext, useContext, useState } from "react";
 import * as grpcWeb from 'grpc-web';
 import { ShootServerClient } from '../../proto/ShootServiceClientPb';
 import { CreateGameRequest, CreateGameResponse, JoinGameRequest, Notification, StatusResponse, TakeSeatRequest } from '../../proto/shoot_pb';
+import { enableMapSet } from "immer"
+
+// Immer - enable support for Map and Set
+enableMapSet()
```

Comment:
Could this go into index.tsx to be applied across the entire app or does it need to be added to each context/component in which it is used?